scripts/globals/abilities/heel.lua

```diff
@@ -26,6 +26,12 @@ end;
 -----------------------------------
 
 function onUseAbility(player,target,ability)
+    local pet = player:getPet()
+
+    if (pet == nil or pet == '') then
```

Comment:
Oh, awesome.  I was afraid of the pet going away between the two.  Good to know!